### PR TITLE
COMMERCE-3311 Added bulk actions for product and orders

### PR DESCRIPTION
**Pull request by alessiorendina**

### CODE REVIEW

**Comment by Commando18** [comment]
This PR needed bulk actions from FE infra team

File: modules/apps/commerce/commerce-order-web/src/main/java/com/liferay/commerce/order/web/internal/display/context/CommerceOrderListDisplayContext.java [hunk 1]
```diff
@@ -55,8 +62,35 @@ public CommerceOrderListDisplayContext(
...
+
+		ThemeDisplay themeDisplay =
+			_commerceOrderRequestHelper.getThemeDisplay();
+
+		PortletURL portletURL = PortletURLFactoryUtil.create(
+			_commerceOrderRequestHelper.getRequest(), portletDisplay.getId(),
+			themeDisplay.getPlid(), PortletRequest.ACTION_PHASE);
+
+		portletURL.setParameter("redirect", themeDisplay.getURLCurrent());
+		portletURL.setParameter(ActionRequest.ACTION_NAME, "editCommerceOrder");
```
Comment (dsitu) [inline modules/apps/commerce/commerce-order-web/src/main/java/com/liferay/commerce/order/web/internal/display/context/CommerceOrderListDisplayContext.java:79]: Hey @alecsloan @Commando18 I think this should be matching the `mvc.command.name` right?

```suggestion
		portletURL.setParameter(ActionRequest.ACTION_NAME, "/commerce_order/edit_commerce_order");
```

**Review by dsitu — CHANGES_REQUESTED** [review]

File: modules/apps/commerce/commerce-order-web/src/main/java/com/liferay/commerce/order/web/internal/display/context/CommerceOrderListDisplayContext.java (hunk 1 above)
Comment (Commando18) [reply modules/apps/commerce/commerce-order-web/src/main/java/com/liferay/commerce/order/web/internal/display/context/CommerceOrderListDisplayContext.java]: Did it, thanks!

**Comment by Commando18** [comment]
ci:test:sf

**Comment by Commando18** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-order-web/src/main/java/com/liferay/commerce/order/web/internal/display/context/CommerceOrderListDisplayContext.java [hunk 2]
```diff
@@ -55,6 +62,34 @@ public CommerceOrderListDisplayContext(
...
+		ThemeDisplay themeDisplay =
+			_commerceOrderRequestHelper.getThemeDisplay();
+
+		PortletURL portletURL = PortletURLFactoryUtil.create(
+			_commerceOrderRequestHelper.getRequest(), portletDisplay.getId(),
+			themeDisplay.getPlid(), PortletRequest.ACTION_PHASE);
+
+		portletURL.setParameter("redirect", themeDisplay.getURLCurrent());
+		portletURL.setParameter(
+			ActionRequest.ACTION_NAME, "commerce_order/edit_commerce_order");
```
Comment (dsitu) [inline modules/apps/commerce/commerce-order-web/src/main/java/com/liferay/commerce/order/web/internal/display/context/CommerceOrderListDisplayContext.java:80]: @Commando18 missing the `/` in the front.
```suggestion
			ActionRequest.ACTION_NAME, "/commerce_order/edit_commerce_order");
```

File: modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/display/context/CPDefinitionsDisplayContext.java [hunk 3]
```diff
@@ -142,6 +143,26 @@ public CreationMenu getAccountGroupsCreationMenu() throws PortalException {
...
 
+	public List<DropdownItem> getBulkActionDropdownItems() {
+		List<DropdownItem> bulkActions = new ArrayList<>();
+
+		RenderResponse renderResponse = cpRequestHelper.getRenderResponse();
+
+		ActionURL actionURL = renderResponse.createActionURL();
+
+		actionURL.setParameter(
+			ActionRequest.ACTION_NAME, "editProductDefinition");
```
Comment (dsitu) [inline modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/display/context/CPDefinitionsDisplayContext.java:154]: Here also, I think it should be this, not 100% sure though
```suggestion
			ActionRequest.ACTION_NAME, "/cp_definitions/edit_cp_definition");
```

**Review by dsitu — CHANGES_REQUESTED** [review]

**Comment by marco-leo** [comment]
@andrea-ale-sbarra any plan for this PR?

**Comment by Commando18** [comment]
ci:test:sf

**Comment by Commando18** [comment]
ci:test:relevant

**Comment by Commando18** [comment]
ci:test:relevant

**Comment by marco-leo** [comment]
ci:forward

**Pull request closed without merging**